pdsfile/pds3file/tests/conftest.py

```diff
@@ -16,17 +13,18 @@ def pytest_addoption(parser):
 
 def turn_on_logger(filename):
     LOGGER = pdslogger.PdsLogger(filename)
-    pdsfile.set_logger(LOGGER)
+    pds3file.set_logger(LOGGER)
 
 # We only use use_pickles and use_shelves_only
 @pytest.fixture(scope='session', autouse=True)
 def setup(request):
     mode = request.config.option.mode
     if mode == '1':
```

Comment:
I can never remember which mode is which. Is there some way to use something else on the command line that's more obvious than "1" and "2"?